### PR TITLE
RN fix RO dependencies for 1.3.1 release

### PR DESCRIPTION
**Pull request by raidernick**
-start updating dependencies to remove/change broken ones and replace
with the updated versions

### CODE REVIEW

**Comment by Bornholio** [comment]
Should also update or preferably remove MM dll, if this is a dependency it should come from source not us
Closes https://github.com/KSP-RO/RealismOverhaul/issues/1796

File: RealismOverhaul.netkan [hunk 1]
```diff
@@ -56,15 +55,10 @@
...
-        { "name" : "LazTekSpaceXExploration" },
-        { "name" : "LazTekSpaceXHistoric" },
-        { "name" : "LazTekSpaceXLaunch" },
         { "name" : "NearFutureConstruction" },
-        { "name" : "NearFutureElectric" },
         { "name" : "NearFuturePropulsion" },
         { "name" : "NearFutureSolar" },
         { "name" : "NearFutureSpacecraft" },
-        { "name" : "PorkjetHabitats" },
         { "name" : "RLAContinued" },
```
Comment (Olympic1) [inline RealismOverhaul.netkan]: Use continued mod `RLARecontinued` for 1.3.1

File: RealismOverhaul.netkan [hunk 2]
```diff
@@ -23,30 +23,29 @@
...
         { "name" : "Toolbar" },
         { "name" : "VenStockRevamp"}
     ],
     "suggests"   : [
         { "name" : "DMagicOrbitalScience" },
         { "name" : "KAS" },
         { "name" : "SCANsat" },
-        { "name" : "AIESAerospace-Unofficial" },
         { "name" : "ALCOR" },
         { "name" : "ATKPropulsionPack" },
```
Comment (Olympic1) [inline RealismOverhaul.netkan]: Not updated since 1.0.5

File: RealismOverhaul.netkan [hunk 3]
```diff
@@ -23,30 +23,29 @@
...
         { "name" : "AtomicAge" },
```
Comment (Olympic1) [inline RealismOverhaul.netkan]: Not updated since 1.0.5 but compatibility set to 1.1.3

File: RealismOverhaul.netkan [hunk 4]
```diff
@@ -23,30 +23,29 @@
         { "name" : "B9-PWings-Fork" },
         { "name" : "ConnectedLivingSpace" },
         { "name" : "DeadlyReentry" },
-        { "name" : "HangarExtender" },
+        { "name" : "HangerExtenderExtended" },
         { "name" : "KSP-AVC" },
         { "name" : "MechJeb2" },
         { "name" : "ProceduralFairings" },
         { "name" : "ProceduralFairings-ForEverything" },
```
Comment (Olympic1) [inline RealismOverhaul.netkan]: Last release was for 1.2.2 and maintained by RO team

File: RealismOverhaul.netkan (hunk 1 above)
Comment (raidernick) [reply RealismOverhaul.netkan]: changed

File: RealismOverhaul.netkan (hunk 2 above)
Comment (raidernick) [reply RealismOverhaul.netkan]: i'm having the author update this tomorrow

File: RealismOverhaul.netkan (hunk 3 above)
Comment (raidernick) [reply RealismOverhaul.netkan]: removed

File: RealismOverhaul.netkan (hunk 4 above)
Comment (raidernick) [reply RealismOverhaul.netkan]: will look into updating or removing

File: RealismOverhaul.netkan (hunk 2 above)
Comment (raidernick) [reply RealismOverhaul.netkan]: It's now updated so this is resolved

File: RealismOverhaul.netkan (hunk 4 above)
Comment (raidernick) [reply RealismOverhaul.netkan]: I have no permissions for this and the people who do are awol so before release I may remove this.

**Comment by raidernick** [comment]
@PhineasFreak done

**Pull request merged**